litellm/utils.py

```diff
@@ -8602,6 +8602,9 @@ def handle_bedrock_stream(self, chunk):
             # cohere mapping
             elif "text" in chunk_data:
                 text = chunk_data["text"]  # bedrock.cohere
+            # mistral mapping
+            elif "outputs" in chunk_data:
+                text = chunk_data['outputs'][0]['text']
```

Comment:
What happens when `outputs` is not a single item list? I have not seen it *not* be a single item list, but this code would throw an IndexError if it was empty. Likewise, we would be missing deltas if the list had more than 1 delta.

Have you found any documentation about this API? (I haven't.)